src/main/java/org/fogbowcloud/app/api/http/controllers/JobController.java

```diff
@@ -60,6 +60,7 @@ public JobController(FileSystemStorageService storageService, JobService jobServ
         List<JDFJob> syncList = new ArrayList<>();
         for(JDFJob job : list) {
             JDFJob syncJob = IguassuApplication.jobSynchronizer.synchronizeJob(job);
+            this.jobService.updateJob(job);
```

Comment:
Not sure we need to refactor it now (or just drop a comment) ... this synchronizedJob name is misleading. Is it an updateJobState? Also, if we rename it to updateXPTO, there is no need to return a reference